src/main/resources/db/migration/V1_5__create_courthouse_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE moj_courthouse (
+  moj_crt_id                INTEGER                         GENERATED ALWAYS AS IDENTITY
+, courthouse_name           CHARACTER VARYING               NOT NULL
```

Comment:
we should probably have a unique constraint on courthouse_name

Reply:
Should it not be courthouse_name and code?